src/types/reader.types.ts

```diff
@@ -39,5 +39,6 @@ export enum StorageKey {
 
 export interface Metadata {
   url: string,
-  type: string
+  type: string,
+  original: boolean
```

Comment:
this is nice for karatsuba, but if someone wants to self host carrot, they probably won't want this

Reply:
an alternative is to just add a 'scalations group' field

Reply:
Yeah that makes sense